utils/build-script

```diff
@@ -2018,6 +2027,11 @@ iterations with -O",
     parser.add_argument(
         "--enable-tsan-runtime",
         help="enable Thread Sanitizer on the swift runtime")
+    parser.add_argument(
+        "--enable-lsan",
+        help="enable Leaks Sanitizer for swift tools",
```

Comment:
Hrm. I don't have strong opinions about it if it's going to be a pain to change. It's pretty consistently Leak across llvm currently, but I also don't think it's necessarily a big deal.